Update links

loicreynier opened the pull request.

Kazhnuz commented:
Wow, thanks a lot for this work !

Yeah, I kinda agree flathub links are kinda better, as it give to people clicking the link a good overview of the app (with screenshot and all, which aren't always here on a repository's readme), and access to installing them easily (and access to the source code repository are still possible from the flathub page if they want most of the time, so it's all good imo).

For archived apps, I think they can be removed, it's better to give focus to apps that have activity and can be updated with future plateform evolution.

loicreynier commented:
Removed archive apps as mentioned above and updated more links by

1. removing Circle tags from apps that are not part of it anymore
2. used Flathub links when possible

Also ran [`markdown-link-check`](https://github.com/tcort/markdown-link-check) to be sure all the links are working.

Kazhnuz commented:
Looks good to me ! Thank you !
